lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,133 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+<div class="calc-main">
+    <a title="${_('Open Calculator')}" href="#" role="button" aria-controls="calculator_wrapper" aria-expanded="false" class="calc">${_("Open Calculator")}</a>
```

Comment:
I talked briefly with @cptvitamin about this and he recommends using `buttons` for form-like actions, not necessarily always within a `form`. Also, Chris Coyier points out towards the bottom that if you're going to use `role="button"` on an `a`, you'll need to add in the additional JS to handle the spacebar "clicks" that go with `buttons`.